doc/api/crypto.md

```diff
@@ -1804,6 +1804,16 @@ added: v15.6.0
 
 The issuer identification included in this certificate.
 
+### `x509.issuerCertificate`
+<!-- YAML
+added: REPLACEME
+-->
+
+* Type: {X509Certificate}
+
+The issuer certificate (if known). Will be `undefined` if the issuer
+certificate is not available.
```

Comment:
This suggests a difference between "not known" and "not available". I imagine they're the same thing, though?

```suggestion
* Type: {X509Certificate|undefined}

The issuer certificate. if the issuer certificate is not known, this property is
`undefined`.
```